notebooks/exploratory/lyl-5-polesie-superclasses-pylandstat.ipynb

```diff
@@ -0,0 +1,1836 @@
+{
```

Comment:
<p>Just a small typo in 'Damaged bog'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>